From 7f023bd4e2bee9c022b91f8cffc7144879824420 Mon Sep 17 00:00:00 2001
From: REDACTED
Date: Sat, 10 Feb 2018 22:35:28 +0000
Subject: [PATCH] Report extra counters
 encoding).

---
diff --git a/runtime/LibFuzzer/Fuzzer/FuzzerTracePC.cpp b/runtime/LibFuzzer/Fuzzer/FuzzerTracePC.cpp
index 5e9f9f2..18527d0 100644
--- a/runtime/LibFuzzer/Fuzzer/FuzzerTracePC.cpp
+++ b/runtime/LibFuzzer/Fuzzer/FuzzerTracePC.cpp
@@ -134,6 +134,10 @@ void TracePC::PrintModuleInfo() {
   }
   if (size_t NumClangCounters = ClangCountersEnd() - ClangCountersBegin())
     Printf("INFO: %zd Clang Coverage Counters\n", NumClangCounters);
+
+  // FIXME: We should upstream this
+  if (size_t NumExtraCounters = ExtraCountersEnd() - ExtraCountersBegin())
+    Printf("INFO: %zd Extra Counters\n", NumExtraCounters);
 }
 
 ATTRIBUTE_NO_SANITIZE_ALL
-- 
2.16.2

